x-pack/plugins/security_solution/scripts/endpoint/endpoint_agent_runner/elastic_endpoint.ts

```diff
@@ -134,21 +166,23 @@ export const enrollEndpointHost = async (): Promise<string | undefined> => {
 
     await execa(`multipass`, agentEnrollArgs);
 
-    const runAgentCommand = `multipass exec ${vmName} --working-directory /home/ubuntu/${vmDirName} -- sudo ./elastic-agent \&>/dev/null`;
+    const runAgentCommand = `ssh ubuntu@${vmInfo.info[vmName].ipv4[0]} -o StrictHostKeyChecking=no -i multipass_ssh_key -tt nohup bash -c 'sudo /home/ubuntu/${vmDirName}/elastic-agent &>/dev/null &'`;
```

Comment:
using `ssh` now to start the endpoint due to issues with using `&` with `multipass exec`.

Reply:
could you try to use `install` instead of `enroll`? I believe it better mimics the behavior on the user machines and potentially should allow us to remove the need to manually run endpoint via ssh

Reply:
this is running, not installing. The install happen earlier.

FYI: when I tried `install` command to the agent, my VM had crapped out because of disk space. It might work now that I am deleting the `.tar.gz` file after extraction, but not sure.

If you are ok with it, can we try in a subsequent PR?

Reply:
I think we could increase the size of the VM disk if it cannot install due to the limited space with `--disk`. 
I'm fine with doing it in the follow up PR, I just thought it would be easier to have one command handling the installation and starting the agent and also we wouldn't need then the whole setup of ssh

Reply:
ohhh. I did not think of it that way.  Then yes, I will try that again to see where I get with it. But: will put this PR on hold since I need to get to other task that are of higher priority. 

thanks for the feedback. I'll set this back to draft.

Reply:
Ok. using `install` works after I increased the disk space on the VM to 8GB (this was needed on my Mac that is running intel chip). I have reverted the code around use of SSH.